packages/nodes-base/nodes/Twist/CommentDescription.ts

```diff
@@ -0,0 +1,623 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const commentOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'comment',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Add',
+				value: 'add',
+				description: 'Adds a new comment to a thread.',
+			},
+			{
+				name: 'Remove',
+				value: 'remove',
+				description: 'Remove a comment',
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Get information about a comment',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Get all comments',
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update a comment',
+			},
+		],
+		default: 'add',
+		description: 'The operation to perform.',
+	},
+] as INodeProperties[];
+
+export const commentFields = [
+	/*-------------------------------------------------------------------------- */
+	/*                                comment:add                                 */
+	/* ------------------------------------------------------------------------- */
+		{
+		displayName: 'Workspace ID',
+		name: 'workspaceId',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'getWorkspaces',
+		},
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'add',
+				],
+				resource: [
+					'comment',
+				],
+			},
+		},
+		required: true,
+		description: 'The id of the workspace.',
+	},
+	{
+		displayName: 'Thread ID',
+		name: 'threadId',
+		type: 'string',
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'add',
+				],
+				resource: [
+					'comment',
+				],
+			},
+		},
+		required: true,
+		description: 'The id of the thread.',
+	},
+	{
+		displayName: 'Content',
+		name: 'content',
+		type: 'string',
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'add',
+				],
+				resource: [
+					'comment',
+				],
+			},
+		},
+		required: true,
+		description: 'The content of the comment.',
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		placeholder: 'Add Field',
+		default: {},
+		displayOptions: {
+			show: {
+				resource: [
+					'comment',
+				],
+				operation: [
+					'add',
+				],
+			},
+		},
+		options: [
+						{
```

Comment:
Indentation needs fixing, also for all others: 227, 247, 254, 261, 414, 418, etc.